Android/Io.OpenCensus/source/Xamarin.Io.OpenCensus.OpenCensusContribGrpcMetrics/Xamarin.Io.OpenCensus.OpenCensusContribGrpcMetrics.csproj

```diff
@@ -0,0 +1,103 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!--
+  https://montemagno.com/converting-xamarin-libraries-to-sdk-style-multi-targeted-projects/
+    
+  -->
+    <PropertyGroup>
+        <AssemblyName>Xamarin.Io.OpenCensus.OpenCensusContribGrpcMetrics</AssemblyName>
+
+        <MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
+        <MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
+        <AndroidUseLatestPlatformSdk>False</AndroidUseLatestPlatformSdk>
+        <AndroidUseIntermediateDesignerFile>True</AndroidUseIntermediateDesignerFile>
+        <AndroidResgenFile>Resources\Resource.designer.cs</AndroidResgenFile>
+
+        <TargetFrameworks>MonoAndroid81</TargetFrameworks>
+        <AndroidCodegenTarget>XAJavaInterop1</AndroidCodegenTarget>
+         
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>        
+        <Description>Bindings for Io.OpenCensus.OpenCensusContribGrpcMetrics package (Google  GRPC  dependency)</Description>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <IsBindingProject>true</IsBindingProject>
+        <AndroidClassParser>class-parse</AndroidClassParser>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <!-- 
+        nuget packaging
+        -->
+        <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+        <PackageId>Xamarin.Io.OpenCensus.OpenCensusContribGrpcMetrics</PackageId>
+        <PackageVersion>0.12.3</PackageVersion>
+        <Title>Xamarin.Io.OpenCensus.OpenCensusContribGrpcMetrics</Title>
+        <Summary>
+            Google GRPC Stub bindings for Xamarin
+        </Summary>
+        <Description>
+            Bindings for Google's GRPC Stub package (Google Play Services dependency)
+        </Description>
+        <Owners>Microsoft</Owners>
+        <Authors>Microsoft</Authors>
+        <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+        <PackageLicenseUrl>
+        https://go.microsoft.com/fwlink/?linkid=2102972&amp;clcid=0x409
```

Comment:
Just trim off the `&amp;clcid=0x409` from the url's, not needed

Reply:
Done.

Reply:
Done